packages/server/src/migrations/migrate.ts

```diff
@@ -696,15 +696,11 @@ function buildCodingTable(result: SchemaDefinition): void {
       { name: 'system', type: 'UUID', notNull: true },
       { name: 'code', type: 'TEXT', notNull: true },
       { name: 'display', type: 'TEXT' },
+      { name: 'isSynonym', type: 'BOOLEAN' },
     ],
     indexes: [
       { columns: ['id'], indexType: 'btree', unique: true },
       { columns: ['system', 'code'], indexType: 'btree', unique: true, include: ['id'] },
-      {
-        columns: ['system', { expression: "to_tsvector('english'::regconfig, display)", name: 'display' }],
-        indexType: 'gin',
-        where: 'display IS NOT NULL',
-      },
```

Comment:
This index is no longer in use, and was replaced by the one below it in the definition